src/SmartReader/TimeToReadCalculator.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace SmartReader
+{
+    internal static class TimeToReadCalculator
+    {
+        private static readonly Dictionary<string, int> charactersMinute = new()
+        {
+            { "Arabic", 612 },
+            { "Chinese", 255 },
+            { "Dutch", 978 },
+            { "English", 987 },
+            { "Finnish", 1078 },
+            { "French", 998 },
+            { "German", 920 },
+            { "Hebrew", 833 },
+            { "Italian", 950 },
+            { "Japanese", 357 },
+            { "Polish", 916 },
+            { "Portuguese", 913 },
+            { "Swedish", 917 },
+            { "Slovenian", 885 },
+            { "Spanish", 1025 },
+            { "Russian", 986 },
+            { "Turkish", 1054 }
+        };
+
+        // based on http://iovs.arvojournals.org/article.aspx?articleid=2166061
+
+        public static TimeSpan Calculate(Article article)
+        {
+            if (string.IsNullOrEmpty(article.TextContent))
+            {
+                return TimeSpan.Zero;
+            }
+
+            int weight = GetWeight(article);
+
+            int letterCount = article.Element?.TextContent.Count(x => x != ' ' && !char.IsPunctuation(x)) ?? 0;
```

Comment:
We may want to consider calculating the time to read on the normalized text content instead of the element's text content. This changes the text values by +/- one minute.

Reply:
Just to be clear, by normalized text content do you mean the text returned by the Converter method? Or do you have in mind something like Unicode normalization?